client/css/index.css

```diff
@@ -109,11 +109,17 @@ nav{
     justify-content: space-around;
     align-items: center;
     list-style: none;
+    margin-top : 5px;
+}
+
+.current {
+    font-weight: bolder;
+    font-size : 30px;
 }
 
 .navStyle{
```

Comment:
In the following commit, change this class name to be `navHeadings`
and also please add space after the name